rasa/nlu/featurizers/dense_featurizer/mitie_featurizer.py

```diff
@@ -71,17 +71,18 @@ def process_training_example(
 
             self._set_features(example, sequence_features, sentence_features, attribute)
 
-    def process(self, message: Message, attribute: Text = TEXT, **kwargs: Any) -> None:
-        if attribute not in DENSE_FEATURIZABLE_ATTRIBUTES:
-            return
-
-        mitie_feature_extractor = self._mitie_feature_extractor(**kwargs)
-        tokens = message.get(TOKENS_NAMES[attribute])
-        sequence_features, sentence_features = self.features_for_tokens(
-            tokens, mitie_feature_extractor
-        )
+    def process(self, message: Message, **kwargs: Any) -> None:
+        for attribute in DENSE_FEATURIZABLE_ATTRIBUTES:
+            mitie_feature_extractor = self._mitie_feature_extractor(**kwargs)
```

Comment:
this line should be outside the loop. No need to instantiate it every iteration.